### PR TITLE
Reader Stream Refresh: Fix header spacing nits

### PR DESCRIPTION
**Pull request by jancavan**
This fixes: https://github.com/Automattic/wp-calypso/issues/8990

**Before:**
screenshot 2016-11-02 13 40 34

**After:**
screenshot 2016-11-02 13 36 17

### CODE REVIEW

**Comment by fraying** [comment]
Perfection! You have made me a very happy pixel pusher.

File: client/blocks/reader-post-card/style.scss [hunk 1]
```diff
@@ -108,6 +108,11 @@ $reader-post-card-breakpoint-small: "( max-width: 550px )";
 	font-size: 13px;
 }
 
+.reader-post-card__byline .reader-avatar,
+.reader-post-card__byline .site-iconm {
```
Comment (blowery) [inline client/blocks/reader-post-card/style.scss]: extra `m` on `.site-icon` ?

**Comment by jancavan** [comment]
Noticed more misalignment issues, cleaned up this block of code as well: https://github.com/Automattic/wp-calypso/pull/9103/commits/886c49b6e18ad08159d18da51553f043f6baf075

screenshot-2016-11-02-14 09 32

**Comment by fraying** [comment]
❤️ LOVE IT ❤️

**Pull request merged**